src/h5web/dataset-visualizer/VisDisplay.tsx

```diff
@@ -2,15 +2,20 @@ import React from 'react';
 import { HDF5Value, HDF5SimpleShape, HDF5Dataset } from '../providers/models';
 import { Vis } from './models';
 import { RawVis, ScalarVis, MatrixVis, LineVis } from '../visualizations';
+import { useValue } from '../providers/hooks';
 
 interface Props {
-  vis: Vis;
+  vis: Vis | undefined;
```

Comment:
`vis?`

Reply:
Si tu veux éviter ça tu peux aussi checker pour `activeVis` dans `DatasetVisualizer`. C'est un peu redondant, mais c'est plus proche de la réalité.

Reply:
Chose to check for `activeVis` in `DataVisualizer`